serializers.go

```diff
@@ -43,6 +44,11 @@ func Serialize(s Serializer, req *prompb.WriteRequest) ([][]byte, error) {
 			labels[string(model.LabelName(l.Name))] = string(model.LabelValue(l.Value))
 		}
 
+		if len(prometheusWhitelist) > 0 && !prometheusWhitelist[labels["__name__"]] {
```

Comment:
The length of `prometheysWhitelist` doesn't change each iteration, so remembering that fact outside of the loop is a little bit more efficient.